tests/float.rs

```diff
@@ -2,9 +2,14 @@ use nom::branch::alt;
 use nom::bytes::complete::tag;
 use nom::character::streaming::digit1 as digit;
 use nom::combinator::{map, map_res, opt, recognize};
+use nom::error::ErrorKind;
```

Comment:
Note: I am not a maintainer.

This file seems to be focused on ASCII float parsing.  The other file might be more appropriate for these tests

Reply:
Thanks for your comments!